compiler/quilt/tools/util.py

```diff
@@ -101,3 +106,29 @@ def sub_files(path, invisible=False):
         files = [x for x in files if not x.startswith('.')]
 
     return files
+
+
+def is_identifier(string):
+    """Check if string could be a valid python identifier
+
+    :param string: string to be tested
+    :returns: True if string can be a python identifier, False otherwise
+    :rtype: bool
+    """
+    # cached by python
+    val = re.match(r'^[a-zA-Z_]\w*$', string) and not keyword.iskeyword(string)
```

Comment:
Let's get rid of keyword checking. Sounds like we're planning to support them in the future, and we don't reject them right now, so not much point in doing it.

Reply:
I've gotten rid of `is_identifier()` and just kept `is_nodename()` with the changes you prescribed.